pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch/SkylineBatch.csproj

```diff
@@ -336,6 +337,9 @@
       <HintPath>..\..\..\..\Shared\Lib\Newtonsoft.Json.dll</HintPath>
       <EmbedInteropTypes>False</EmbedInteropTypes>
     </Reference>
+    <Reference Include="pwiz.Common">
+      <HintPath>..\..\..\..\Shared\Common\bin\x64\Release\pwiz.Common.dll</HintPath>

```

Comment:
You should remove this reference to "pwiz.Common.dll"
This project is only supposed to have a reference to "CommonUtil".